modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2703,6 +2703,28 @@ def explode(self, axis: Union[int, Axis], func: Callable) -> "PandasDataframe":
             partitions, new_index, new_columns, row_lengths, column_widths
         )
 
+    def to_pandas_in_remote_function(self) -> "PandasDataframe":
```

Comment:
```suggestion
    def combine(self) -> "PandasDataframe":
```
or `to_single_partition`

Reply:
ping